test/guard_intf_test.cpp

```diff
@@ -55,59 +63,38 @@ TEST_F(TestGuardRecord, CreateGuardRecord)
 {
```

Comment:
Please add documentation for each test

Reply:
Generally test case don't need any explicit documentation as the names of the test cases are quite self explanatory itself. Wherever any special logic and/or conclusion is needed are been mentioned. So I don't think any exclusive documentation is needed for any test case.